src/main/java/org/spongepowered/api/Client.java

```diff
@@ -53,6 +56,11 @@
      */
     Optional<ClientPlayerController> getPlayerController();
 
+    @Override
+    default void sendMessage(ChatType type, Text message) {
```

Comment:
I removed this method earlier. It delegated to the PlayerController, so it would send a message to the client.